internal/backend/sessions/sessionworkflows/callbacks_events.go

```diff
@@ -47,6 +50,9 @@ func (w *sessionWorkflow) subscribe(wctx workflow.Context) func(context.Context,
 
 func (w *sessionWorkflow) unsubscribe(wctx workflow.Context) func(context.Context, sdktypes.RunID, string) error {
 	return func(ctx context.Context, rid sdktypes.RunID, signalID string) error {
+		_, span := w.startCallbackSpan(ctx, "unsubscribe")
```

Comment:
Why do we need span? IMO performance metrics are enough. If you want to find a bottleneck - run a profiler.

Reply:
span gets displayed appropiately in the trace. that way we can see exactly what takes time as part of the trace.